src/SonataMediaBundle.php

```diff
@@ -51,11 +55,11 @@ public function boot(): void
     public function registerFormMapping(): void
     {
         FormHelper::registerFormTypeMapping([
-            'sonata_media_type' => 'Sonata\MediaBundle\Form\Type\MediaType',
-            'sonata_media_api_form_media' => 'Sonata\MediaBundle\Form\Type\ApiMediaType',
-            'sonata_media_api_form_doctrine_media' => 'Sonata\MediaBundle\Form\Type\ApiDoctrineMediaType',
-            'sonata_media_api_form_gallery' => 'Sonata\MediaBundle\Form\Type\ApiGalleryType',
-            'sonata_media_api_form_gallery_item' => 'Sonata\MediaBundle\Form\Type\ApiGalleryItemType',
+            'sonata_media_type' => MediaType::class,
+            'sonata_media_api_form_media' => ApiMediaType::class,
+            'sonata_media_api_form_doctrine_media' => ApiDoctrineMediaType::class,
+            'sonata_media_api_form_gallery' => ApiGalleryType::class,
+            'sonata_media_api_form_gallery_item' => ApiGalleryItemType::class,
```

Comment:
This line needs a use: `use Sonata\MediaBundle\Form\Type\ApiGalleryItemType;`

Sorry for the multiples reviews, a lot of changes make it hard to review it fast

Reply:
No problem, I understand ;)